user-script/popup/user-script.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const hostsInput =  document.querySelector("#hosts");
+const codeInput =  document.querySelector("#code");
+
+const defaultHosts = "*://*.org/*";
+const defaultCode = "document.body.innerHTML = '<h1>This page has been eaten<h1>'";
```

Comment:
I found that this didn't work on https://mozila.org/ for me, could be the runAt bug. Something like an `alert('Hello from user-script)` worked just fine though.

Reply:
That's odd, it works for me on mozilla.org (specifically https://www.mozilla.org/en-US/). You do have to reload the page though.

Reply:
Does it work on other pages, like developer.mozilla.org?

Reply:
Hmm, working now. Blame user error, I did something wrong :)